documentation/library_guide/onedpl_gsg.rst

```diff
@@ -131,7 +131,7 @@ fit into the square *S*. Random number generation is performed in scalar manner
 
     float estimated_pi;
     {
-        sycl::queue q(sycl::gpu_selector{});

```

Comment:
```gpu_selector``` is deprecated: https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:device-selector